src/dnfile/base.py

```diff
@@ -154,6 +161,8 @@ class MDTableRow(abc.ABC):
     _struct_enums: Dict[str, Tuple[str, Type[enum.IntEnum]]]         # also enum.IntEnum subclassA
     _struct_lists: Dict[str, Tuple[str, str]]                        # also Metadata table name
 
+    _loaded = LoadState.Unloaded
```

Comment:
is this meant to be a class variable? as its declared at the class scope, the property is shared across all instances of the class, rather than a specific instance.

Reply:
Good catch, thanks